packages/cli/internal/pkg/cli/zipfile/compress_to_tmp.go

```diff
@@ -25,6 +27,14 @@ func CompressToTmp(srcPath string) (string, error) {
 }
 
 func writeToZipRecursive(writer *zip.Writer, rootPath string) error {
+	// Expand home directory path
+	usr, _ := user.Current()
```

Comment:
We probably don't want to ignore this error as it will cause issues if the customer is using home path and the Current() function fails but we also don't want to throw an error if the prefix doesn't exist in the path definition.

Reply:
Changed it to use the existing `DetermineHomeDir` which provides error handling.